src/py2spack/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""py2spack package.
+
+This package allows converting standard python packages to Spack packages.
+"""
+
+from .main import SpackPyPkg
+
+__version__ = "0.0.1"
+
+__all__ = ["SpackPyPkg"]
```

Comment:
Don't forget that our coding conventions (Google coding conventions in this case) build on top of PEP 8, the most basic language style convention which says that the special/magic/dunder module variables should be defined before any regular import:    https://peps.python.org/pep-0008/#module-level-dunder-names
```suggestion
__all__ = ["SpackPyPkg"]
__version__ = "0.0.1"

from .main import SpackPyPkg
```
The rationale for this is to set the module metadata before anything else, to avoid possible errors when importing code relying in this metadata being already defined.